scripts/src/logseq/tasks/lang.clj

```diff
@@ -1,33 +1,68 @@
 (ns logseq.tasks.lang
   "Tasks related to language translations"
-  (:require [logseq.rewrite-clj :as rewrite-clj]
+  (:require [logseq.tasks.rewrite-clj :as rewrite-clj]
             [clojure.set :as set]
             [logseq.tasks.util :as task-util]))
 
 (defn- get-dicts
   []
-  (dissoc (rewrite-clj/var-sexp ["dicts" "src/main/frontend/dicts.cljs"])
+  (dissoc (rewrite-clj/var-sexp "src/main/frontend/dicts.cljs" "dicts")
           :tongue/fallback))
 
+(defn- get-non-en-shortcuts
+  []
+  (nth (rewrite-clj/var-sexp "src/main/frontend/modules/shortcut/dict.cljs" "dict")
+       3))
+
+;; unnecessary complexity :(
+(defn- decorate-namespace [k]
+  (let [n (name k)
+        ns (namespace k)]
+    (keyword (str "command." ns) n)))
+
+(defn- get-en-shortcut-dicts
+  []
+  (->> (rewrite-clj/var-sexp
+        "src/main/frontend/modules/shortcut/config.cljs"
+        "all-default-keyboard-shortcuts")
+       (map (fn [[k v]] (vector (decorate-namespace k) (:desc v))))
```

Comment:
I don't think having commands decorated is a good design decision. I see needless complexity in our codebase because of it and it makes it difficult to map what is occurring in a couple namespaces including frontend.modules.shortcut.dict and frontend.modules.shortcut.config. I held off on refactoring this but I think it would be simpler for us to have the full keywords in shortcut/core.cljs e.g. `:command.editor/bold` instead of `:editor/bold`. I realize users depend on the latter so we could still support the old syntax just just for the user config

Reply:
Agreed that we should switch to full keywords in shortcut/core.cljs.